spec/index.bs

```diff
@@ -3765,16 +3765,16 @@ interface GPUQueue {
     void signal(GPUFence fence, GPUFenceValue signalValue);
 
     void writeBuffer(
-        ArrayBuffer sourceData,
-        GPUSize64 sourceOffset,
-        GPUBuffer destination,
-        GPUSize64 destinationOffset,
-        GPUSize64 size);
+        GPUBuffer buffer,
+        GPUSize64 bufferOffset,
+        ArrayBuffer data,
+        optional GPUSize64 dataOffset = 0,
+        optional GPUSize64 size);
 
     void writeTexture(
-      ArrayBuffer sourceData,
-      GPUTextureDataLayout sourceLayout,
       GPUTextureCopyView destination,
+      GPUTextureDataLayout dataLayout,
```

Comment:
it seems to me that the `dataLayout` needs to go after the array buffer, since it applies to the contents of that buffer (sorry about the churn on ordering!)

Reply:
No worries, done.